news/2021-10-04-halloween-fanart-contest.md

```diff
@@ -0,0 +1,48 @@
+
+Onwards to the rules!
+
+## Rules
+
+- **By submitting your artwork to this contest, you grant us permission to use your artwork in the main menu of the osu! client should it win, and to use it in promotional material for both the game (osu!) and the contest (Halloween 2021 Fanart Contest).** You retain all rights to your creation and you can sell it and distribute it however you like — it is your work, after all. This is the fine print. Make sure you always read this in future contests you enter, as not all of them do we what we do here!
+- **All entries MUST be themed around any of the osu! (or community) mascots wrapped up in Halloween shenanigans.** Keep to the theme! This is a Halloween contest, it should be about Halloween at a first glance.
+- **All entries MUST be suitable for all-ages viewing.** Witches wear ROBES and ROBES generally cover things. Halloween is usually pretty good for this, but still!
+- **SUBMIT ONLY YOUR OWN WORK. REDRAWING AND COMPOSITE WORKS ARE NOT ALLOWED.** Plagiarism sucks. Don't do it. Someone in the comments will call you out for it as has happened before and they're going to do more damage to your reputation than us removing your entry from the voting. Use only your own work — brushes and the like are fine, but that's it! Don't redraw someone else's work.
+- **ARTIST ATTRIBUTION (names, social media handles, etc) MUST BE LIMITED TO NO MORE THAN THREE LINES TOTAL AND CONFINED TO EITHER THE BOTTOM LEFT OR BOTTOM RIGHT CORNERS OF YOUR ENTRY.** Being proud of your work is good, just do it in a reasonably small spot of your work. Please understand that we're not very fond of anything ad-related in osu!, and attribution that is blaring enough can be considered as one.
+- **Entries MUST meet the resolution requirements for a main-menu piece — namely 2732x1536. THIS IS CHECKED BY A SCRIPT AND ENTRIES THAT DO NOT MATCH THIS REQUIREMENT WILL BE DISCARDED AUTOMATICALLY.** This is the #1 reason for entries not making it to voting phase. MAKE SURE YOUR ENTRY MEETS THE EXACT RESOLUTION REQUIREMENTS ABOVE. Check your work after you've exported it and BEFORE you submit it. The ghoul butler that processes entries in this way is brutal and will discard your work without mercy, so make sure it's the right resolution!
+- **Artists should use the custom templates provided ([PSD template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.psd?2017) / [PNG template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.png?2017)) when composing their piece as a guideline to understand where the osu! cookie will be placed on the main menu and during the voting process.** Anything you draw that is inside the dotted circle won't be seen during voting phase, because we super-impose an osu! cookie over all entries during voting phase to simulate what they're going to look like on the main menu. Please accommodate for this in your drawing!
```

Comment:
```suggestion
- **Artists should use the custom templates provided ([PSD template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.psd?2017) / [PNG template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.png?2017)) when composing their piece as a guideline to understand where the osu! cookie will be placed on the main menu and during the voting process.** Anything you draw that is inside the dotted circle won't be seen during the voting phase, because we super-impose an osu! cookie over all entries during the voting phase to simulate what they're going to look like on the main menu. Please accommodate for this in your drawing!
```